src/constants/update-details-journeys.js

```diff
@@ -1,7 +1,9 @@
 module.exports = Object.freeze({
-  UPDATE_ADDRESS: "address",
+  UPDATE_ADDRESS: "address-only",
   UPDATE_NAMES_DOB: "names-dob",
-  UPDATE_NAME: "name",
-  UPDATE_NAME_ADDRESS: "name-address",
+  UPDATE_GIVEN_NAMES: "given-names-only",
+  UPDATE_FAMILY_NAME: "family-name-only",
+  UPDATE_GIVEN_NAME_ADDRESS: "given-names-and-address",
+  UPDATE_FAMILY_NAMES_ADDRESS: "family-name-and-address",
```

Comment:
The middleware is unhappy as it's trying to access these values using the below, which I think is right. So might need an update here.

```suggestion
  UPDATE_GIVEN_NAMES_ADDRESS: "given-names-and-address",
  UPDATE_FAMILY_NAME_ADDRESS: "family-name-and-address",
```